### PR TITLE
Patch travis timeout

### PR DESCRIPTION
**Pull request by gavraz**

### CODE REVIEW

File: tests/late_nodes/test_delayed.py [hunk 1]
```diff
@@ -60,6 +60,13 @@ def new_client_in_namespace(name_space, setup_bootstrap, cspec, num):
     return client_info
 
 
+# this is a path for travis's 10m timeout limit
+# we reached the timeout because epochDuration happened to be greater than 10m
+def sleep_and_print(duration):
+    time.sleep(duration/2) # wait half
+    print("Half of duration passed") # print something for travis
```
Comment (beckmani) [inline tests/late_nodes/test_delayed.py]: Please print something more meaningful  (Duration of what?)

File: tests/late_nodes/test_delayed.py [hunk 2]
```diff
@@ -82,10 +89,12 @@ def test_add_delayed_nodes(init_session, setup_bootstrap, save_log_on_exit):
...
         clients[i] = new_client_in_namespace(ns, setup_bootstrap, cspec, numToAdd)
         print("Added client batch ", i, clients[i].pods[i]['name'])
-        time.sleep(epochDuration)
+        sleep_and_print(epochDuration)
 
     print("Done adding clients. Going to wait for two epochs")
-    time.sleep(2*epochDuration) # wait two more epochs
+    # wait two more epochs
+    sleep_and_print(epochDuration)
+    sleep_and_print(epochDuration)
```
Comment (beckmani) [inline tests/late_nodes/test_delayed.py]: Why you need to call it twice?

File: tests/late_nodes/test_delayed.py (hunk 1 above)
Comment (gavraz) [reply tests/late_nodes/test_delayed.py]: Fixed

File: tests/late_nodes/test_delayed.py (hunk 2 above)
Comment (gavraz) [reply tests/late_nodes/test_delayed.py]: Fixed

**Review by beckmani — APPROVED** [review]

**Pull request merged**